frontend/app/src/components/status/NotificationIndicator.vue

```diff
@@ -42,7 +42,7 @@ const click = () => {
   emit('click');
 };
 
-const { hasRunningTasks } = useTasks();
+const { hasRunningTasks } = storeToRefs(useTasks());
```

Comment:
I was under the impression that `storeToRefs` only returned computed refs/refs, it also handles getters, but what about actions?

I think the impression I had was because normal of the normal `toRefs` we used in the beginning.